lightly/api/api_workflow_compute_worker.py

```diff
@@ -414,11 +429,10 @@ def get_compute_worker_run_info(
     def compute_worker_run_info_generator(
         self, scheduled_run_id: str
     ) -> Iterator[ComputeWorkerRunInfo]:
-        """
-        Yields information about a compute worker run
+        """Pulls information about a compute worker run continuously.
 
         Polls the compute worker status every 30s.
-        If the status changed, it will yield a new ComputeWorkerRunInfo.
+        If the status changed, an update pops up.
```

Comment:
```suggestion
        If the status changed, an update is yielded.
```

Or is there any particular reason not to use yield?

Reply:
Using yield in passive voice just doesn't sound right to me.